docs/installation.md

```diff
@@ -503,6 +503,8 @@ sudo cp runc /usr/bin/runc
 
 #### Setup CNI networking
 
+For CNI networking, the `cni` build tag needs to be used. Otherwise `netavark` is used as default since version 5 of Podman.
```

Comment:
The whole section should be removed IMO or at least heavily reworded, we no longer support it upstream. If distro users enable it they are own their own.